CHANGELOG.md

```diff
@@ -311,6 +311,16 @@ Pragmas and options
 
 * Option `--count-clusters` is now on by default when Agda was built with ICU support (Cabal flag `enable-cluster-counting`).
 
+* New pragma `INJECTIVE_FOR_INFERENCE`, which treats functions as injective for inferring implicit arguments. For example:
```

Comment:
1. Why it cannot be enhance `INJECTIVE`, does it break anything?
2. Is this related to denominational/propositional injectiveness, which was mentioned here? https://github.com/agda/agda/issues/4106#issuecomment-534904561